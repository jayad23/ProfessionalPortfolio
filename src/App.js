
import { BrowserRouter, Routes, Route } from "react-router-dom";

// Styles
import './App.css';

//Importations
import Home from "./Views/Home";
import About from "./Views/About";
import WorkHistory from "./Views/WorkHistory";
import Skills from "./Views/Skills";
import MainLayOut from "./Custom/MainLayOut";
import Form from "./components/Form";
import { Blog } from "./Views/Blog";

//Provider
import { ProfesionalProvider } from "./Context/ProfesionalInfoContext";
import Academic from "./Views/Academic";

function App() {
  return (
    <BrowserRouter>
      <ProfesionalProvider>
        <MainLayOut>
          <Routes>
            <Route path="/" exact element={<Home/>}/>
            <Route path="/about" exact element={<About/>}/>
            <Route path="/academic" exact element={<Academic/>}/>
            <Route path="/workhistory" exact element={<WorkHistory/>}/>
            <Route path="/skills" exact element={<Skills/>}/>
            <Route path="/blog" exact element={<Blog/>}/>
            <Route path="/contact" exact element={<Form/>}/>
          </Routes>
        </MainLayOut>
      </ProfesionalProvider>
    </BrowserRouter>
  );
 
}

export default App;
