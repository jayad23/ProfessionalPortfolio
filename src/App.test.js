// import { render, screen } from '@testing-library/react';


// test('renders learn react link', () => {
//   render(<App />);
//   const linkElement = screen.getByText(/Portfolio/i);
//   expect(linkElement).toBeInTheDocument();
// });

import React from 'react'
import ReactDom from 'react-dom'
import renderer from 'react-test-renderer'
import App from './App';

test('App snapShot test', ()=>{
  const component = renderer.create(<App/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot()
})

// it('renders without crashing', ()=>{
//   const div = document.createElement('div');
//   ReactDom.render(<App/>, div)
// })