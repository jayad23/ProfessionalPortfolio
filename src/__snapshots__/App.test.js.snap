// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapShot test 1`] = `
<div>
  <div
    className="icon"
  >
    <i
      className="far fa-envelope"
      onClick={[Function]}
    />
  </div>
  <div
    className="container-home"
  >
    <div
      className="home-image"
    >
      <img
        alt=""
        src="self.jpg"
      />
    </div>
    <div
      className="home-text"
    >
      <h3
        onMouseEnter={[Function]}
      >
        Hello
      </h3>
      <h4>
        I am Kike Vanegas
      </h4>
      <p>
         
        <b
          className="text-iter styles_typicalWrapper__1_Uvh"
        />
      </p>
    </div>
    <aside
      className="home-social"
    >
      <ul>
        <li>
          <a
            className="fab fa-facebook-f"
            href="https://www.facebook.com/kkvz23/"
            rel="noreferrer"
            target="_blank"
          />
        </li>
        <li>
          <a
            className="fab fa-twitter"
            href="https://twitter.com/KkVngz23"
            rel="noreferrer"
            target="_blank"
          />
        </li>
        <li>
          <a
            className="fab fa-youtube"
            href="https://www.youtube.com/channel/UCCzn5vab6xL1-Vwdzk8Meeg"
            rel="noreferrer"
            target="_blank"
          />
        </li>
        <li>
          <a
            className="fab fa-linkedin"
            href="https://www.linkedin.com/in/jaidervanegas23/"
            rel="noreferrer"
            target="_blank"
          />
        </li>
        <li>
          <a
            className="fab fa-instagram"
            href="https://www.instagram.com/kkvngz23/"
            rel="noreferrer"
            target="_blank"
          />
        </li>
        <li>
          <a
            className="fab fa-whatsapp"
            href="https://api.whatsapp.com/send?phone=573196149019&text=Hello, Kike. I am interested in your work..."
            rel="noreferrer"
            target="_blank"
          />
        </li>
      </ul>
    </aside>
  </div>
</div>
`;
